website/meta/sidebars.json

```diff
@@ -91,6 +91,7 @@
                     { "text": "EntityRuler", "url": "/api/entityruler" },
                     { "text": "Lemmatizer", "url": "/api/lemmatizer" },
                     { "text": "Morphologizer", "url": "/api/morphologizer" },
+                    { "text": "Multi-label TextCategorizer", "url": "/api/multilabel_textcategorizer" },
```

Comment:
I don't think we should have a separate page for this (also, all the other pages here use the class name, so this one would have to be `MultiLabel_TextCategorizer`). Since it's a subclass of the `TextCategorizer` with _very_ minor modifications, I think it's fine to document this together with the `TextCategorizer`, basically as a sepcialised variant.

Reply:
Ah, ok, I should merge the information from `/api/multilabel_textcategorizer` back into the normal textcat one then.